code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -632,11 +632,11 @@
 		else if(!user.IsStunned())
 			target.Stun(0.5 SECONDS)
 	else
-		if(target.IsSlowed() && target.get_active_hand())
+		if(target.IsSlowed() && target.get_active_hand() && !IS_HORIZONTAL(user) && !HAS_TRAIT(target, TRAIT_WIELDED))
```

Comment:
You want to check for the trait on the item in their hand, not the person

Reply:
Ahhh I thought it was applied to the user, not the item, will do!